flows/routers/tests/tests.go

```diff
@@ -347,7 +349,7 @@ func HasPattern(env utils.Environment, text types.XText, pattern types.XText) ty
 //
 // @test has_number(text)
 func HasNumber(env utils.Environment, text types.XText) types.XValue {
-	return testNumber(env, text, types.XNumberZero, isNumberTest)
+	return testNumber(env, text, types.XNumberZero, types.XNumberZero, isNumberTest)
```

Comment:
reworked these number test functions to take 2 test args so that `HasNumberBetween` can use the same code path